.yamllint

```diff
@@ -1,6 +1,9 @@
 ---
 extends: default
 
+ignore: |
+  operator-pipeline-images/tests/data/operator-repo
```

Comment:
Do you think we could remove this line and make a repo compliant with a linting?

Reply:
I expect to move the operator_repo module, and therefore the test data out of this repo at some point.
I don't think it's worth the effort to try to fix all the yamls.